### PR TITLE
feat: add the methods of serializing and deserializing fixvec

### PR DESCRIPTION
**Pull request by Keith-CY**

### CODE REVIEW

File: src/fixvec/fixvec.spec.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    if (exception) {
+      expect(() => serializeFixvec(source)).toThrow(exception)
+    } else {
+      const actual = serializeFixvec(source)
+      expect(actual).toBe(expected)
+    }
+  })
+})
+
+describe('Test deserialize array', () => {
```
Comment (duanyytop) [inline src/fixvec/fixvec.spec.ts]: Maybe test deserialize fixvec

File: src/fixvec/index.ts [hunk 2]
```diff
@@ -1 +1,46 @@
...
+    throw new TypeError(`Expect ${fixvec} to be an array`)
+  }
+
+  fixvec.forEach(item => {
+    assertIsHexStr(item)
+  })
+
+  const data = fixvec.map(item => (item.length % 2 ? `0${item.slice(2)}` : item.slice(2)))
+
+  const length = data[0]?.length ?? 0
```
Comment (duanyytop) [inline src/fixvec/index.ts]: Length is an uint32 data

Comment (Keith-CY) [reply src/fixvec/index.ts]: The `length` here is used to make sure all items have the same length and is not included in the serialization.

File: src/fixvec/fixvec.spec.ts (hunk 1 above)
Comment (Keith-CY) [reply src/fixvec/fixvec.spec.ts]: fixed in https://github.com/Keith-CY/molecule-javascript/pull/15/commits/92429b8897265d9d2d4dbe2fb662d457400c2ea9

File: src/fixvec/index.ts (hunk 2 above)
Comment (duanyytop) [reply src/fixvec/index.ts]: OK

**Pull request merged**